vllm/v1/worker/xpu_worker.py

```diff
@@ -176,3 +177,14 @@ def init_device(self):
         # Construct the model runner
         self.model_runner = XPUModelRunner(  # type: ignore
             self.vllm_config, self.device)
+
+
+def _check_if_gpu_supports_dtype(torch_dtype: torch.dtype):
```

Comment:
I prefer keep this in xpu.py since we will remove `xpu_worker.py` in the future.

Reply:
moved to xpu.py